### PR TITLE
Added support for PHP Tidy extension.

### PR DESCRIPTION
**Pull request by schkovich**
[Tidy](http://php.net/manual/en/book.tidy.php) is a binding for the Tidy HTML clean and repair utility which allows you to not only clean and otherwise manipulate HTML documents, but also traverse the document tree.

### CODE REVIEW

**Comment by jippi** [comment]
:+1:

File: manifests/extension/tidy.pp [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+#
+# No variables
+#
+# === Examples
+#
+#  include php::extension::tidy
+#
+# === Authors
+#
+# Christian "Jippi" Winther <jippignu@gmail.com>
```
Comment (igalic) [inline manifests/extension/tidy.pp]: you might want to fix up the author to say your name here…

File: manifests/extension/tidy.pp [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+#
+# === Copyright
+#
+# Copyright 2012-2013 Christian "Jippi" Winther, unless otherwise noted.
```
Comment (igalic) [inline manifests/extension/tidy.pp]: @jippi the copyright years would need updating in every file :P

File: manifests/extension/tidy.pp (hunk 1 above)
Comment (schkovich) [inline manifests/extension/tidy.pp]: All I did is copy/paste and find/replace. :) 

At best I could add my name below Christian's but even that is doubtful.

**Comment by igalic** [comment]
@schkovich can you please "rebase" this (i.e.: fix the copyright year) and then we'll merge it.

**Comment by schkovich** [comment]
done @igalic

File: manifests/extension/tidy.pp (hunk 1 above)
Comment (jippi) [inline manifests/extension/tidy.pp]: @schkovich add your name if you want, I just initially copy+pasted the header from another module - don't care much, its a MIT plugin :)

Comment (schkovich) [inline manifests/extension/tidy.pp]: ok. done.

**Comment by jippi** [comment]
Thanks!

**Pull request merged**